.gitmodules

```diff
@@ -1,10 +1,10 @@
 [submodule "tools/extract-xiso"]
 	path = tools/extract-xiso
-	url = https://github.com/mborgerson/extract-xiso.git
+	url = https://github.com/xboxdev/extract-xiso.git
 [submodule "lib/net/lwip"]
 	path = lib/net/lwip
 	url = git://git.savannah.nongnu.org/lwip.git
```

Comment:
Please put the lwip update in a separate commit, possibly even a separate PR.

If lwip breaks, we'd have to revert *all* the changes, including the Windows fix (which might also just be a workaround?).

Same goes for other stuff like changing the upstream for the other submodules, or fixing code in cxbe.
However, they are probably smaller changes which are not as critical

Only group those changes which are absolutely necessary to reach your goal (PR). And each unrelated step towards that goal should also be separate (commit).